src/components/Modal/Modal.js

```diff
@@ -0,0 +1,26 @@
+import "./Modal.scss";
+import styles from './Modal.scss';
+
+const Modal = ({ selectedMovie, setModalIsOpen }) => {
+  console.log(selectedMovie)
+
+  const handleClick = (isOpen) => {
+    setModalIsOpen(isOpen);
+  }
+
+  return (
+    <div className={styles.modal}>
```

Comment:
Stoked to see you trying to use SCSS - something is up here tho. I would expect some regular classnames here - accessing the styles about from a different file doesn't seem right